packages/cli/src/databases/repositories/execution.repository.ts

```diff
@@ -717,6 +731,8 @@ export class ExecutionRepository extends Repository<ExecutionEntity> {
 
 			if (query.order?.stoppedAt === 'DESC') {
 				qb.orderBy({ 'execution.stoppedAt': 'DESC' });
+			} else if (query.order?.top) {
+				qb.orderBy(`(CASE WHEN execution.status = '${query.order?.top}' THEN 0 ELSE 1 END)`);
```

Comment:
Just to be sure, have we tested this on all db systems?

Reply:
There is a test [here](https://github.com/n8n-io/n8n/pull/9453/files#diff-f80cf5df958cc26e37cbb046b7ed170f267728e9a4527c33e572bda5695bb552R438) passing on [all DBs](https://github.com/n8n-io/n8n/actions/runs/9414927037).